sys/linux/dev_kvm_amd64.txt

```diff
@@ -471,6 +491,33 @@ kvm_sev_snp_launch_finish {
 	pad1		array[const[0, int64], 4]
 }
 
+kvm_tdx_capabilities {
+	supported_attrs			int64
```

Comment:
Should this be flags that include TDX_TD_ATTR_SEPT_VE_DISABLE?

Reply:
kvm_tdx_capabilities is an output parameter, does it matter which values it returns?